cmd/kubeadm/app/constants/constants.go

```diff
@@ -266,7 +266,7 @@ const (
 	DefaultEtcdVersion = "3.4.3-0"
 
 	// PauseVersion indicates the default pause image version for kubeadm
-	PauseVersion = "3.1"
+	PauseVersion = "3.2"
```

Comment:
a non-blocking wonder what this will do.
my guess it will only apply to new clusters and existing clusters will have to opt-in manually for this sandbox update for dockershim, as kubeadm upgrade does not manage this at all.

cc @rosti @bart0sh

Reply:
Yes. That's because we supply that image only with the `--pod-infra-container-image` when we generate the kubelet env file. However, we don't regenerate that file upon upgrade (only on initial kubelet setup). Hence, users who only do upgrades would not get the new pause version.
But then again, `--pod-infra-container-image` only works with Docker...

Reply:
yeah, annoyingly this is otherwise handled in config of the CRI implementation and not specified by CRI / kubelet. we'll need to update those as well.

Reply:
https://github.com/containerd/cri/pull/1399